darts/timeseries.py

```diff
@@ -880,6 +880,41 @@ def is_within_range(self,
         index = self.time_index()
         return index[0] <= ts <= index[-1]
 
+
+    def map(self,
+            fn: Callable[[float], float],
```

Comment:
For the type hints you can use `Callable[[np.number], np.number]` to be consistent with type check we do upon TimeSeries creation.

Reply:
Done.